app/plugins.go

```diff
@@ -84,3 +89,129 @@ func InitPlugins() {
 		p.OnConfigurationChange()
 	}
 }
+
+func ActivatePlugins() {
+	if Srv.PluginEnv == nil {
+		l4g.Error("plugin env not initialized")
+		return
+	}
+
+	plugins, err := Srv.PluginEnv.Plugins()
+	if err != nil {
+		l4g.Error("failed to start up plugins: " + err.Error())
+		return
+	}
+
+	for _, plugin := range plugins {
+		err := Srv.PluginEnv.ActivatePlugin(plugin.Manifest.Id)
+		if err != nil {
+			l4g.Error(err.Error())
+		}
+		l4g.Info(fmt.Sprintf("Activated %v plugin", plugin.Manifest.Id))
```

Comment:
What's the style for log messages? Upper or lowercase first word?

Also, pretty sure the log functions have built-in formatting:

```go
l4g.Info("Activated %v plugin", plugin.Manifest.Id)
```

If not, they should. :P